demos/common/python/models/model.py

```diff
@@ -15,15 +15,20 @@
 """
 
 import logging
-
+import numpy as np
 
 class Model:
-    def __init__(self, ie, model_path):
+    def __init__(self, ie, model_path, reverse_input_channels=None, mean_values=None, scale_values=None):
```

Comment:
```suggestion
    def __init__(self, ie, model_path, reverse_input_channels, mean_values, scale_values):
```
`Model`s' children don't have default values (and I think they shouldn't).

Reply:
The default values are necessary here, otherwise the models constructor with unified API (like segmentation) is failed in `super().__init__(ie, model_path)` part.